<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>13</y>
      <width>751</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_adminID_tag">
       <property name="text">
        <string>管理员：</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox_aricon">
         <property name="text">
          <string>中央空调</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_changeAircon">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="3">
        <widget class="QSpinBox" name="spinBox_hotHighLimit"/>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="spinbox_defaultTemp"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="combobox_defaultWind">
         <item>
          <property name="text">
           <string>低</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>中</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>高</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_defaultTemp">
         <property name="text">
          <string>缺省温度</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_serveNum">
         <property name="text">
          <string>服务对象</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinbox_feeRate"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_defaultWind">
         <property name="text">
          <string>缺省风速</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_tempLimit">
         <property name="text">
          <string>~</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox_hotLowLimit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_feeRate">
         <property name="text">
          <string>费率</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_hotPattern">
         <property name="text">
          <string>制热</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QSpinBox" name="spinBox_serveNum"/>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_coldPattern">
         <property name="text">
          <string>制冷</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_coldLowLimit"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_tempLimit_2">
         <property name="text">
          <string>~</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="spinBox_coldHighLimit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_checkState">
       <property name="text">
        <string>查看状态</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser_state"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_quit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
